protocol/agents.json

```diff
@@ -0,0 +1,118 @@
+{
+    "agents": [
+        {
+            "identifier": "ably-js",
+            "emitters": [
+                "ably-js"
+            ],
+            "type": "sdk"
+        },
+        {
+            "identifier": "node",
+            "emitters": [
+                "ably-js"
+            ]
+        },
+        {
+            "identifier": "react-native",
+            "versioned": false,
+            "emitters": [
+                "ably-js"
+            ],
+            "type": "wrapper"
+        },
+        {
+            "identifier": "ably-cocoa",
+            "emitters": [
+                "ably-cocoa"
+            ],
+            "type": "sdk"
+        },
+        {
+            "identifier": "iOS",
+            "emitters": [
+                "ably-cocoa"
+            ],
+            "type": "os"
+        },
+        {
+            "identifier": "tvOS",
+            "emitters": [
+                "ably-cocoa"
+            ],
+            "type": "os"
+        },
+        {
+            "identifier": "watchOS",
+            "emitters": [
+                "ably-cocoa"
+            ],
+            "type": "os"
+        },
+        {
+            "identifier": "macOS",
+            "emitters": [
+                "ably-cocoa"
+            ],
+            "type": "os"
+        },
+        {
+            "identifier": "ably-java",
+            "emitters": [
+                "ably-java"
+            ],
+            "type": "sdk"
+        },
+        {
+            "identifier": "android",
+            "emitters": [
+                "ably-java"
+            ],
+            "type": "os"
+        },
+        {
+            "identifier": "ably-ruby",
+            "emitters": [
+                "ably-ruby"
+            ],
+            "type": "sdk"
+        },
+        {
+            "identifier": "ruby",
+            "emitters": [
+                "ably-ruby"
+            ]
+        },
+        {
+            "identifier": "ably-ruby-rest",
+            "versioned": false,
+            "emitters": [
+                "ably-ruby"
+            ],
+            "type": "wrapper"
+        }
+    ],
+    "ablyLibMappings": {
+        "js-web": "ably-js/${version} browser",
```

Comment:
All the agent values that appear here should be in the `agents` array too (i.e. `browser`, `nativescript`, `uwp` etc)

Reply:
done 23e8f2c